library/prelude/path.red

```diff
@@ -121,8 +121,6 @@ def J/eq
   | k=1 → refl
   ]
 
-quit
-
 def symmd (A : 𝕀 → type) (p : (i : 𝕀) → A i) : pathd (symm^1 _ A) (p 1) (p 0) =
```

Comment:
@favonia This raises some stupid `Dead` exception.  I guess this is what you are already looking at?